server/src/main/java/com/decathlon/ara/cartography/SquashExporter.java

```diff
@@ -27,8 +54,117 @@ public String getFormat() {
     }
 
     @Override
-    public byte[] generate(List<FunctionalityDTO> functionalities) {
-        // TODO
+    public byte[] generate(List<FunctionalityDTO> functionalities, Map<String, String> requiredInfos) {
+        String projectNameInSquash = requiredInfos.get(PROJECT_NAME);
+        String userWhichMakesTheImport = requiredInfos.get(USER);
+
+        HSSFWorkbook workbook = new HSSFWorkbook();
+        Sheet sheet = workbook.createSheet("REQUIREMENT");
+
+        this.createHeaderRow(sheet);
+        Date importDate = new Date();
+        int idx = 1;
```

Comment:
Same comment as above

Reply:
idx is the abbrev of index which is the array's index